config/routes.rb

```diff
@@ -923,6 +923,13 @@
       get '/user/new' => 'single_provider_users#new', as: :new_single_provider_user
       post '/user' => 'single_provider_users#create', as: :create_single_provider_user
 
+      namespace :bulk_upload, path: '/bulk-upload' do
+        resource :provider_users_details, path: '/provider-users-details', only: %i[new create]
+        resource :permissions, only: %i[edit update]
```

Comment:
Good question. I'm not sure what the best solution is for this. The reason why I changed this to 'edit' and 'update' is because on the permissions form it is also possible to edit the ProviderUsers details (first name, last name, email). 

This may be a sign that the ProviderUserDetailsController and PermissionsController should be merged? But then the disadvantage is that we have a bigger controller and lose the ability to map controllers against the steps in the flow.

Reply:
Agreed to leave as is for now